libs/dotcms-webcomponents/src/components/contenttypes-fields/dot-form/services/dot-upload.service.ts

```diff
@@ -121,9 +121,9 @@ export class DotUploadService {
     private errorHandler(response: any, status: number): DotHttpErrorResponse {
         let message = '';
         try {
-            message = response.message || fallbackErrorMessages[status];
+            message = response.message ?  response.message : fallbackErrorMessages[status];
```

Comment:
What's the diff here?

![image](https://user-images.githubusercontent.com/751424/135156280-a4f4fcf8-dfe4-43ba-bee7-a8a4dbe4b4c6.png)

Reply:
There's no difference 😆 , I forgot to change that. I'll do it!